vm/src/obj/objint.rs

```diff
@@ -566,6 +567,11 @@ impl PyInt {
         !self.value.is_zero()
     }
 
+    #[pymethod(name = "__sizeof__")]
+    fn sizeof(&self, _vm: &VirtualMachine) -> usize {
```

Comment:
Maybe use `self.value.bits()` and then round up to the nearest byte for this.

Reply:
something like `(self.value.bits() as f64 / 8.).ceil() as usize` ?